mining-network-prop/README.md

```diff
@@ -12,24 +12,51 @@
 [Why Bitcoin Mining Pools Aren't Incentivized to Broadcast Blocks Quickly](https://bitcoinmagazine.com/articles/why-bitcoin-mining-pools-aren-t-incentivized-to-broadcast-blocks-quickly-1475249510/) | 10 min | spy mining |
 [Majority is not Enough: Bitcoin Mining is Vulnerable](https://www.cs.cornell.edu/~ie53/publications/btcProcFC.pdf) | 30 min | selfish mining |
 [Demystifying The Selfish Mining Bet](https://eklitzke.org/demystifying-the-selfish-mining-bet) | 10 min | selfish mining |
-[Analysis of Bitcoin Pooled Mining Reward Systems](https://sites.cs.ucsb.edu/~rich/class/cs293b-cloud/papers/bitcoin-pool.pdf) (Intro, Pay-per-share, Pay-per-last-N-shares, Attack Vectors sections) | 30 min | mining pools |
-[The problem with ASIC Boost](http://www.mit.edu/~jlrubin//public/pdfs/Asicboost.pdf) | 15 min | asic boost |
-[Compact Blocks/FIBRE background](https://podcast.chaincode.com/2020/03/12/matt-corallo-6.html) | 32 min | FIBRE, history, podcast, *optional* |
+[Mining pools](https://en.wikipedia.org/wiki/Mining_pool) | 15 min | mining pools |
+[Overview of the covert AsicBoost allegation](https://blog.bitmex.com/an-overview-of-the-covert-asicboost-allegation-2/) | 15 min | asic boost |
 
 ## Discussion Questions
 
-1. How can changes to Bitcoins p2p protocol affect mining? (Advances in Block Propagation)
-1. How does the FIBRE network work and what were the motivations to iterate on the original Relay Network? (Relay Networks and FIBRE)
-1. How do pools give work assignments to client miners? Do all miners search the same block candidates?
-1. What are 'shares' and how do they help facilitate pooled mining?
-1. Consider the following scenario: Segwit2x didn’t happen (or miners didn’t throw in support for it) and UASF (BIP148) ended up causing a chain split on August 1st, 2017, with minority mining support (e.g. ~10-20%). What would likely happen as a result?
+### Mining and Consensus
+
+1. Why would some miners use parts of the version field as a nonce? What effect did this have on nodes?
```

Comment:
Maybe that would fit better into the section on ASICBoost at the bottom?

Reply:
I'd like to keep this as is. As far as I understand, miners have used the version field as extra nonce space independent of ASICBoost.